tests/e2e/csi_snapshot_basic.go

```diff
@@ -261,10 +261,21 @@ var _ = ginkgo.Describe("Volume Snapshot Basic Test", func() {
 			}
 		}()
 
+		if isQuotaValidationSupported && supervisorCluster {
+			_, _, pvc_storagePolicyQuotaBefore, _, pvc_storagePolicyUsageBefore, _ =
+				getStoragePolicyUsedAndReservedQuotaDetails(ctx, restConfig,
+					storageclass.Name, namespace, pvcUsage, volExtensionName, islatebinding)
+
+			_, _, snap_storagePolicyQuotaBefore, _, snap_storagePolicyUsageBefore, _ =
+				getStoragePolicyUsedAndReservedQuotaDetails(ctx, restConfig,
+					storageclass.Name, namespace, snapshotUsage, snapshotExtensionName, islatebinding)
+		}
+
 		ginkgo.By("Create PVC")
 		pvclaim, persistentVolumes, err := createPVCAndQueryVolumeInCNS(ctx, client, namespace, labelsMap, "",
```

Comment:
This PVC creation will fail if you are creating it with WFFC binding mode, because inside this util we are checking PVC and PV to reach Bound state. If it is not in a Bound state, we are failing this util